app/package/controllers.py

```diff
@@ -133,10 +131,6 @@ def delete_data_package(publisher, package):
         status_db = Package.change_status(publisher, package, PackageStateEnum.deleted)
         if status_acl and status_db:
             return jsonify({"status": "OK"}), 200
-        if not status_acl:
-            raise Exception('Failed to change acl')
-        if not status_db:
-            raise Exception('Failed to change status')
     except Exception as e:
```

Comment:
Again: no need for this try/except. Let's do standard exception handling as per http://flask.pocoo.org/docs/0.12/patterns/apierrors/ (i.e. register an error handler for Exception that does the app.logger.error(e) and the return handler ...